# The Malaria vector software ecosystem

Welcome to the Malaria vector software ecosystem homepage!

This site highlights a collection of software tools and resources for malaria vector research. It is a work in progress and we welcome contributions from the community.

We aim to build a training resource for the community. If you have any ideas, tutorials or training material that you would like to share, please get in touch.


--- 

```{tableofcontents}
```



# Learning outcomes

Whilst working through the course, we would like participants to gain confidence in the following:

- Understand the concepts behind WGS, RNA-Sequencing and Amplicon sequencing. 
- Understand the utility of workflow managers (Snakemake, Nextflow) in bioinformatics. 
- Feel equipped with tools to analyse multiple types of omics data - in particular RNA-Sequencing and Amplicon Sequencing. 
- Enhance research into genomic surveillance and novel resistance variant discovery.. 


# Intended audience





# Prerequisites

This course is intended to be an introduction to bioinformatics and tools in the malaria vector ecosystem, but users should already have had some experience with programming and usage of the command line. 

---

#### [PAMCA-MalariaGEN - Training course in data analysis for genomic surveillance of African malaria vectors](https://anopheles-genomic-surveillance.github.io/home.html)

This training course is intended to be complementary to the PAMCA-MalariaGEN training, and so we recommend completing the PAMCA-MalariaGEN training course, either before or after this course. The PAMCA course is a brilliant and comprehensive introduction to whole-genome sequencing, python, and population genomics, and as a result, we will only cover these topics briefly in this training course. 


#### Bioinformatic data skills

We recommend reading the first few chapters of the Bioinformatic data skills prior to beginning the course, more information is [here]() 